frontend/services/entities/mdm.ts

```diff
@@ -82,6 +85,14 @@ export interface IGetMdmCommandResultsResponse {
   results: IMdmCommandResult[];
 }
 
+export interface IGetSetupExperienceScriptResponse {
+  id: number;
+  team_id: number;
```

Comment:
```suggestion
  team_id: number | null;
```

On the backend, this has the same shape as a normal script. For no team scripts, the team id is `null` in API responses.

Reply:
ok will make this change. although I think the API should return 0 for no team as we usually do.